Automattic/wp-calypso
# SDK: Remove legacy SDK files

#### Changes proposed in this Pull Request

Remove `bin/sdk-cli.js`, `bin/sdk/*`, and mentions of those files. These are considered legacy, and superseded by tooling provided by the `@automattic/calypso-build` package.

I opted to remove `docs/sdk.md` altogether. It will be superseded by a `README.md` in `calypso-build` (tracking issue: #31679).

#### Testing instructions

Grep to verify there are no more references to:
- `bin/sdk-cli`
- `npm run sdk`
- `docs/sdk.md`
